style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  font-family: "Poppins", sans-serif;
```

Comment:
I can see you are trying to use the Poppins font, but unfortunately, you forgot to import it 😃
In order for custom fonts to work, you have to import them at the top of your `style.css` file or in your `index.html`.
Kindly go to [Google fonts](https://fonts.google.com/) and import the Poppins font family from there.